app/controllers/SubmitPoaController.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import play.api.libs.json.JsValue
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+import java.net.URI
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class SubmitPoaController @Inject()(cc: MessagesControllerComponents,
+                                    requestHandlerController: RequestHandlerController) extends FrontendController(cc) {
+
+  def interceptSubmitForErrorUser(): Action[AnyContent] = Action.async { implicit request =>
+
+    val bodyOption: Option[JsValue] = request.body.asJson
+    val InvalidJsonError = BadRequest("Invalid JSON in the UpdateIncomeSourceRequest body")
+    val NoNinoError = BadRequest("Nino not found in the ClaimToAdjustPoaRequest body")
+
+    bodyOption match {
+      case Some(body) =>
+        val ninoOpt = extractNino(body)
+        ninoOpt match {
+          case Some(nino) =>
+            val newRequest =
+              if (nino.toUpperCase.equals("PA000006A")) {
```

Comment:
* add support for the list of Nino's;
* move list of Nino's within application config;